lib/topology/pytest/plugin.py

```diff
@@ -187,19 +185,25 @@ class StepLogger(object):
     This class will log a message and will show the step number and the caller
     name and line number.
     """
-    def __init__(self):
+    def __init__(self, test_case):
         self.step = 0
+        self._test_case = test_case
+        self._logger = logging.get_logger(
+            OrderedDict(
+                [('step', 'step'), ('test_case', self._test_case)]
+            ), category='step'
+        )
 
     def __call__(self, msg):
         self.step += 1
         frame, filename, line_number, function_name, lines, index = \
             stack()[1]
-        log.debug(
+        self._logger.log(
             '>>> [{:03d}] :: {}:{}'.format(
                 self.step, function_name, line_number
             )
         )
-        log.debug(
+        self._logger.log(
             '\n'.join(
                 ['... {}'.format(l) for l in msg.strip().splitlines()]
             )
```

Comment:
Ok, open for comments, how do you @carlos-jenkins and @dajose want the log level shown?